src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -0,0 +1,87 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.ConflictResolution.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+    <!-- The condition on this target causes it to be skipped during design-time builds if
+        the restore operation hasn't run yet.  This is to avoid displaying an error in
+        the Visual Studio error list when a project is created before NuGet restore has
+        run and created the assets file. -->
+  <Target Name="_GetLockFileAssemblies"
+          DependsOnTargets="ComputePrivateAssetsPackageReferences;
+                            _DefaultMicrosoftNETPlatformLibrary"
+          Condition=" '$(DesignTimeBuild)' != 'true' Or Exists('$(ProjectAssetsFile)')">
+    <!-- Essentially a copy of the SDKs RunResolvePublishAssemblies.
```

Comment:
@eerhardt I'm not talking about the deps file here.  This target has nothing to do with that.

All this target is doing is getting the set of runtime assets that "will be loaded from packages" for build.  It's totally reasonable to have that item populated during build and I'm pretty sure there's already a target that does it.

Moreover, ResolvePublishAssemblies shouldn't even exist.  That should just be a filtered set of the items already raised from the lock file.  I'm fine if that work is put off a bit, but it seems like we could at least run the one target (RunResolvePublishAssemblies) during build, and just make sure that we didn't consume its output (by copying) in the build target closure.